src/build.d

```diff
@@ -159,6 +159,7 @@ Command-line parameters
     // A more proper solution would be to redirect DMD's output to this script's
     // output using `std.process`', but it's more involved and the following
     // "just works"
+    version(Posix) // UPDATE: only when ANSII color codes are supported, that is. Don't do this on Windows.
     if (!flags["DFLAGS"].canFind("-color=off") &&
```

Comment:
Wasn't that check already sufficient? You could have disabled colors by setting `DFLAGS` to `"-color=off"`, if I understand the code correctly. My guess is that colors where enabled on Windows by default as our Windows CIs have no trouble displaying them in their UI.